### ISSUE TITLE
Drag and drop option to move the page from 1 folder to other

### ISSUE THREAD

**Issue by spartan4cs** [vivekweb2013/gitnoter#55]
**Is your feature request related to a problem? Please describe.**
changing of path is tedious when organizing your notes

**Describe the solution you'd like**
easyily moving the page from 1 folder to other by drag and drop

**Comment by vivekweb2013** [comment]
yes this feature would make organising notes more easier and will greatly improve user experience.

**Comment by itsdiptabiswas** [comment]
is there anyone working on this issue ?

**Comment by vivekweb2013** [comment]
Hey @Charlygraphy23, you can pick this issue if you want

**Comment by shraf** [comment]
hi, are you still working on this project? can i contribute to this issue? @vivekweb2013

**Comment by vivekweb2013** [comment]
@shraf sure, go ahead and raise a PR 🎉

**Comment by Perruer** [comment]
Since this project has been quiet for a while: drag and drop of notes and folders is in Notewing (https://perruer.github.io/notewing/), my rewrite of BatNoter that works without the batnoter.com server. A move is saved as a single commit, so the file history stays readable. Your existing BatNoter repo opens as is.
